src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -24,12 +22,14 @@ class Focuser(AbstractSerialFocuser):
     """
 
     def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', port=None,
-                 vendor_id=0x0403, product_id=0x6001, *args, **kwargs):
+                 vendor_id=0x0403, product_id=0x6001, calibration_position=-25000, *args, **kwargs):
```

Comment:
Let's name this `zero_position` with a `default of `0` and then do the override when we create an instance
```suggestion
                 vendor_id=0x0403, product_id=0x6001, zero_position=-25000, *args, **kwargs):
```